src/modules/tests/test-data/cat-c/vehicle-checks/__tests__/vehicle-checks.cat-c.reducer.spec.ts

```diff
@@ -0,0 +1,68 @@
+import {
+  initialState,
+  vehicleChecksCatBEReducer,
+} from '../vehicle-checks.cat-c.reducer';
+import {
+  ShowMeQuestionSelected,
+  ShowMeQuestionOutcomeChanged,
+  TellMeQuestionSelected,
+  TellMeQuestionOutcomeChanged,
+} from '../vehicle-checks.cat-c.action';
+import { QuestionResult } from '@dvsa/mes-test-schema/categories/common';
+import { CatBEUniqueTypes } from '@dvsa/mes-test-schema/categories/BE';
+import { cloneDeep } from 'lodash';
+
+describe('Vehicle Checks Cat B+E Reducer', () => {
+
+  describe('SHOW_ME_QUESTION_SELECTED', () => {
+    it('should add the show me question in the payload to the array at the specified index', () => {
+      const newQuestionPayload: QuestionResult = {
+        code: 'S1',
+        description: 'desc',
+      };
+      const state: CatBEUniqueTypes.VehicleChecks = cloneDeep(initialState);
+      const result = vehicleChecksCatBEReducer(state, new ShowMeQuestionSelected(newQuestionPayload, 1));
```

Comment:
should be changed to CatC once reducer changed.

Reply:
changed it